hwdb/70-mouse.hwdb

```diff
@@ -329,6 +329,10 @@ mouse:usb:v046dpc52b:name:Logitech Unifying Device. Wireless PID:1017:
 mouse:usb:v046dp1017:name:Logitech Anywhere MX:
  MOUSE_WHEEL_CLICK_ANGLE=20
 
+# Logitech Anywhere MX 2s
+mouse:usb:v046dp406a:name:Logitech MX Anywhere 2S:
+ MOUSE_WHEEL_CLICK_ANGLE=20
```

Comment:
why not merge this entry with the entry three lines up?

Reply:
situational blindness... will update